src/cli/main.cpp

```diff
@@ -39,13 +48,29 @@ int main(int argc, char* argv[]) {
 
   if (!args.empty() && (std::string_view(args[0]) == "labs")) {
     CLI::App app{"openstudio"};
+    app.name(programName);
```

Comment:
I'll no longer use CLI11_PARSE(argc, argv) so I need to set the app.name manually.